src/main/java/carpet/script/value/EntityValue.java

```diff
@@ -476,6 +476,31 @@ public Value get(String what, Value arg)
             return Value.NULL;
         });
 
+        put("absorption",(e, a)->{
+            if(e instanceof PlayerEntity) return new NumericValue(((PlayerEntity) e).getAbsorptionAmount());
+            return Value.NULL;
+        });
```

Comment:
There's no need to restrict setting absorption level to players only since all living entities have the capability to have absorption points.

Reply:
huh I didnt know that.

Reply:
Umm, no. That method only exists in PlayerEntity.java, not Entity.java. Entities don't have absorption.